1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -414,6 +416,8 @@ def create_infofiles(row, filenames_file, not_ok_file):
                     content_categories_set.add("[[" + kw["category"] + "]]") 
     #print("content_categories_set: \n{}".format(content_categories_set))
     
+    if linne_category:
+        categories += "\n[[Category:Sigvald_Linné]]"
```

Comment:
To differentiate the content category (images of Linné) and the meta category (images by Linné) you could change this one to `[[Category:Images_by_Sigvald_Linné]]`.